observant-otters/forumthing/user/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class ForumUser(AbstractUser):
+    username = models.CharField(unique=True, max_length=50)
+    nickname = models.CharField(default='', max_length=50)
+    email = models.EmailField(default='none@smth.com')
+
+    def __str__(self):  # maybe a better string repr?
+        if self.nickname:
+            return self.nickname
+        return self.username
```

Comment:
You would want to implement `__repr__` if you're making an unambiguous representation of something, especially if it's for debugging. This could probably be a `__repr__`.

You might be interested to know, this function could simply be `return self.nickname or self.username`, as an `or` expression will return the first value in the chain that is truthy, or the last item if none of them do. It could also be `return self.nickname if self.nickname else self.username`.